crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -18,59 +18,66 @@
 
 module Api
   class Upgrade < Tableless
-    def status
-      {
-        crowbar: crowbar_upgrade_status,
-        checks: check
-      }
-    end
+    class << self
+      def status
+        {
+          crowbar: crowbar_upgrade_status,
+          checks: check
+        }
+      end
 
-    def check
-      {
-        sanity_checks: sanity_checks,
-        maintenance_updates_installed: maintenance_updates_installed?,
-        clusters_healthy: clusters_healthy?,
-        compute_resources_available: compute_resources_available?,
-        ceph_healthy: ceph_healthy?
-      }
-    end
+      def check
+        {
+          sanity_checks: sanity_checks,
+          maintenance_updates_installed: maintenance_updates_installed?,
+          clusters_healthy: clusters_healthy?,
+          compute_resources_available: compute_resources_available?,
+          ceph_healthy: ceph_healthy?
+        }
+      end
 
-    def cancel
-      service_object = CrowbarService.new(Rails.logger)
-      service_object.revert_nodes_from_crowbar_upgrade
+      def cancel
+        service_object = CrowbarService.new(Rails.logger)
+        service_object.revert_nodes_from_crowbar_upgrade
 
-      true
-    rescue => e
-      errors.add(:base, e.message)
-      Rails.logger.error(e.message)
+        {
```

Comment:
It's not immediately obvious to me why you are changing the return value here, and the commit message doesn't explain.  It feels like maybe you are combining two logical commits into a single commit without any explanation, which makes it quite a bit harder to review.